install/install.md

```diff
@@ -47,6 +47,16 @@ Use the selector tool below to select your preferred method, packages, and envir
 ## Installation Troubleshooting
 
 ### **Conda Issues**
+<i class="fas fa-info-circle"></i> CUDA 12.0 arm packages don't exist:<br/>
```

Comment:
```suggestion
<i class="fas fa-info-circle"></i> CUDA 12.0 ARM packages are not yet available:<br/>
```

Reply:
ARM capitalization is confusing! Happy to get a clear answer here lol (Company name is `Arm`, logo is `arm` and arch is `ARM` it seems!)

Reply:
The rule I go by is to always capitalize ARM (which really implies ARM SBSA for us, meaning “ARM in the server flavor and not Jetson which is called L4T”) or instead say aarch64 (the proper name of the arch used by conda — less common in our docs).